fr/console-and-shells/input-output.rst

```diff
@@ -1,37 +1,39 @@
-Command Input/Output
-====================
+Entrée/sortie de commande
+=========================
 
 .. php:namespace:: Cake\Console
 .. php:class:: ConsoleIo
 
-CakePHP provides the ``ConsoleIo`` object to commands so that they can
-interactively read user input and output information to the user.
+CakePHP fournit l'objet ``ConsoleIo`` aux commandes afin qu'elles puissent
+lire interactivement les informations d'entrée et de sortie de l'utilisateur.
 
-Command Helpers
-===============
+Helpers (Assistants) de commande
+================================
 
-Command Helpers can be accessed and used from any command, shell or task::
+Les Helpers (Assistants) de commande sont accessibles et utilisables depuis
+n'importe quelle commande, shell ou tâche : ::
 
-    // Output some data as a table.
+    // Affiche des données en tant que tableau.
     $io->helper('Table')->output($data);
 
-    // Get a helper from a plugin.
+    // Récupère un helper depuis un plugin.
     $io->helper('Plugin.HelperName')->output($data);
 
-You can also get instances of helpers and call any public methods on them::
-
-    // Get and use the Progress Helper.
+Vous pouvez aussi récupérer les instances des Helpers et appeler n'importe
+quelle méthode publique dessus : ::
+    // Récupérer et utiliser le ProgressHelper.
     $progress = $io->helper('Progress');
     $progress->increment(10);
     $progress->draw();
 
-Creating Helpers
-================
+Créer des Helpers (Assistants)
+==============================
 
-While CakePHP comes with a few command helpers you can create more in your
-application or plugins. As an example, we'll create a simple helper to generate
-fancy headings. First create the **src/Command/Helper/HeadingHelper.php** and put
-the following in it::
+Alors que CakePHP est fourni avec quelques helpers de commande, vous pouvez
+en créer d'autres dans votre application ou vos plugins. À titre d'exemple,
+nous allons créer un helper simple pour générer des titre élégants.
+Créez d'abord le fichier **src/Command/Helper/HeadingHelper.php** et mettez
+ce qui suit dedans : ::
```

Comment:
You don't need 3 `:` only 2 are required.

Reply:
But when using only 2 : (with the space before) no : is redered at all

Reply:
Odd. Sphinx usually inserts a `:` when `::` is used. For example https://book.cakephp.org/3.0/en/core-libraries/events.html and https://raw.githubusercontent.com/cakephp/docs/3.0/en/core-libraries/events.rst

Reply:
![sublime_text_2018-08-02_19-39-59](https://user-images.githubusercontent.com/9319054/43600891-2b851550-968c-11e8-8e03-68dacaa609b5.png)
As you can see, putting a space before the `::` doesn't automatically add the `:`